### ISSUE TITLE
Video Endpoints

### ISSUE THREAD

**Issue by EdieFarahkuhcoon** [ryanmcgrath/twython#399]
I am trying to update through pip --upgrade and it seems that new video upload endpoint calls are not merged into pypi version

**Comment by michaelhelmick** [comment]
Hi @EdieFarahkuhcoon this will be merged soon. Waiting on another pull request to make a release!
